### PR TITLE
Bugzilla: Config CI-tools 

### PR DESCRIPTION
**Pull request by GeoBK**
Config file modifications for the new bugzilla backporter tool.

### CODE REVIEW

File: clusters/app.ci/assets/bugzilla-backporter.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  spec:
+    lookupPolicy:
+      local: true
+    tags:
+    - name: latest
+      importPolicy:
+        scheduled: true
+      from:
+        kind: DockerImage
+        name: registry.svc.ci.openshift.org/ci/bugzilla-backporter
```
Comment (petr-muller) [inline clusters/app.ci/assets/bugzilla-backporter.yaml:18]: Should we only merge the deployment once we actually have the image?

Comment (GeoBK) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:18]: Yeah! Thats the intention, but will probably have to fix stuff before thats ready.

File: clusters/app.ci/assets/bugzilla-backporter.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            mountPath: /etc/config
+            readOnly: true
+          resources:
+            requests:
+              memory: "2Gi"
+              cpu: "200m"
+        volumes:
+        - name: config
+          configMap:
+            name: config
```
Comment (GeoBK) [inline clusters/app.ci/assets/bugzilla-backporter.yaml]: I wanted to leave a volume mount for the secret file here. I am not really sure if that gets configured here or someplace else. Left this in as a placeholder till then

File: clusters/app.ci/assets/bugzilla-backporter.yaml (hunk 1 above)
Comment (alvaroaleman) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:18]: I think peters point is that the two things need to be seperate PRs, first the build config, second the yamls in here

Comment (GeoBK) [reply clusters/app.ci/assets/bugzilla-backporter.yaml:18]: Oh ok. Got it!

**Comment by alvaroaleman** [comment]
/lgtm cancel
You have to run `make jobs`

**Pull request merged**